searchcore/src/vespa/searchcore/proton/server/feedhandler.cpp

```diff
@@ -547,7 +548,7 @@ FeedHandler::initiateCommit() {
     if (_activeFeedView) {
         using KeepAlivePair = vespalib::KeepAlive<std::pair<CommitResult, DoneCallback>>;
         auto pair = std::make_pair(std::move(commitResult), std::move(onCommitDoneContext));
-        _activeFeedView->forceCommit(_serialNum, std::make_shared<KeepAlivePair>(std::move(pair)));
+        _activeFeedView->forceCommit(CommitParam(_serialNum, false), std::make_shared<KeepAlivePair>(std::move(pair)));
```

Comment:
No need to use explicit false here. Could use _CommitParam(_serialNum)_.

Reply:
As false is a performance optimization, it is better to make the optimization explicit and default to non-optimized.